libz80e/src/debugger/commands/load_register.c

```diff
@@ -0,0 +1,46 @@
+#include "debugger/commands.h"
+#include "debugger/debugger.h"
+#include <ctype.h>
+#include <string.h>
+
+int command_load_register(debugger_state_t *state, int argc, char **argv) {
+	if (argc != 3) {
+		state->print(state, "%s - Load a register with a specified value. Supports hex (i.e. 0x0000) and base-10 (i.e. 12)\n", argv[0]);
+		return 0;
+	}
+
+#define REGISTER(num, len, print) \
+	if (strncasecmp(argv[1], print, len) == 0) {\
+		state->asic->cpu->registers. num = parse_expression(state, argv[2]); \
```

Comment:
Nevermind, I was wrong. I thought z80e shared the same expression parsing code as scas (it does not).